File: beginner/python/1164.py
x = int(input())

for i in range(0,x):
    soma = 0
    n = int(input())

    for j in range(1,n):
        if(n % j == 0):
            soma += j
        
    if(soma == n):
        print("%d eh perfeito" % n)
    else:
        print("%d nao eh perfeito" % n)
    

File: beginner/javascript/1164.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split('\n');

var i, j, x, n, soma;

x = parseInt(linhas.shift());

for(i = 0; i < x; i++){
    soma = 0;
    n = parseInt(linhas.shift());

    for(j = 1; j < n; j++){
        if(n % j == 0){
            soma += j;
        }
    }

    if(soma == n){
        console.log(n + " eh perfeito");
    } else{
        console.log(n + " nao eh perfeito");
    }

}
